cmd/ossyscall.go

```diff
@@ -20,8 +20,17 @@ import (
 	"os"
 	"os/signal"
 	"syscall"
+
+	"github.com/dapr/cli/pkg/print"
 )
 
 func setupShutdownNotify(sigCh chan os.Signal) {
 	signal.Notify(sigCh, syscall.SIGTERM, syscall.SIGINT)
 }
+
+// createProcessGroupID creates a process group id for the current process.
```

Comment:
```suggestion
// createProcessGroupID creates a process group ID for the current process.
```

Also please explain what it does and why.